src/tool.js

```diff
@@ -0,0 +1,39 @@
+exports.genForm = (inputs) => {
+    // Create a blank string to start with
+    var formStr = '';
+    try {
+        inputs.forEach((input) => {
+            // Make sure input has property `type`
+            if (input.type) {
+                // Switch expression handles accepted input `type` props.
+                // toLowerCase makes sure we don't have to handle case sensitivity
+                switch (input.type.toLowerCase()) {
```

Comment:
I don't think you need the switch statement here imagine if you want to validate all of the possibilities.
Also, you could use template strings to make your code cleaner!

Here is an example of the solution that thought to eliminate repetitive code:
you could have an array and push the inputs for that array if it is valid.

`let fields = []`

Example:
```js
  formStr = `<input type="${input.type}" name="${input.name}" id=${input.id}/>`;
  fields.push(formStr);
```

the <`input`> could also be a template string!